server/templates/about.html

```diff
@@ -0,0 +1,61 @@
+{#
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+#}
+{% extends 'base.html' %}
+
+{% set main_id = 'about' %}
+{% set title = 'About Us' %}
+
+{% block head %}
+  <link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['GAE_VERSION'])}} />
+{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <h1>About Data Commons</h1>
+    <p>
+      Publicly available data from open sources (census.gov, NOAA, data.gov,
+      etc.) are vital resources for students and researchers in a variety of
```

Comment:
should we add a larger target users here? students and researches maybe a bit narrowed?

Reply:
Yes, probably. The content on that page could use a lot of work.. I just pulled out the content from our current homepage there for now.